.gitlab-ci.yml

```diff
@@ -165,615 +165,12 @@ default:
     | tee artifacts/benches/$CI_COMMIT_REF_NAME-$CI_COMMIT_SHORT_SHA/::trie::read::small.json'
   - sccache -s
 
-#### stage:                       .pre
```

Comment:
Those changes will of course be reverted before the PR is merged. I've removed the other steps other than `check-polkadot-companion-build` so that the pipeline is fast for development.